src/wrap-store/wrapStore.js

```diff
@@ -94,6 +94,10 @@ export default (store, {
 
     const serializedMessagePoster = withSerializer(serializer)((...args) => port.postMessage(...args));
 
+    const isChrome = /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
```

Comment:
Whatever test we do use should be exported into a util and then imported in these files.